op_crates/web/02_abort_signal.js

```diff
@@ -36,11 +36,20 @@
         throw new TypeError("Illegal constructor.");
       }
       super();
-      this.onabort = null;
-      this.addEventListener("abort", (evt) => {
-        const { onabort } = this;
-        if (typeof onabort === "function") {
-          onabort.call(this, evt);
+      // Dom specification section 8.1.5.1
+      let eventHandler = null;
+      Object.defineProperty(this, 'onabort', {
```

Comment:
In browsers this is installed on the prototype as:

```js
{
  get() {}
  set() {}
  enumerable: true,
  configurable: true,
}
```

We should probably align, which would be a getter and setter in the class?  Just like `aborted` (well which doesn't have a setter)?

Reply:
Good catch, I'll fix this.